python/orca/src/bigdl/orca/common.py

```diff
@@ -158,22 +158,29 @@ def get_ray_context():
         return RayContext.get()
 
 
-def init_orca_context(cluster_mode="local", cores=2, memory="2g", num_nodes=1,
+def init_orca_context(cluster_mode=None, cores=2, memory="2g", num_nodes=1,
                       init_ray_on_spark=False, **kwargs):
     """
     Creates or gets a SparkContext for different Spark cluster modes (and launch Ray services
     across the cluster if necessary).
 
     :param cluster_mode: The mode for the Spark cluster. One of "local", "yarn-client",
-           "yarn-cluster", "k8s-client", "standalone" and "spark-submit". Default to be "local".
+           "yarn-cluster", "k8s-client" and "standalone". Default to be None.
```

Comment:
Default to be None and in this case there is supposed to be an existing SparkContext in your application.